11ty/definitions/pt_PT/capacitismo.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Capacitismo / Aptidismo
```

Comment:
I can't find any meaning for "Aptidismo".

Reply:
Was unsure what the correct Portuguese word is. Googling I found capacitismo and aptidismo on https://pt.qaz.wiki/wiki/Kyriarchy

Reply:
I see. Also found that reference there. Actually, the only one when googling. Never heard the term before and am missing references. I would stay with "Capacitismo" only.

Reply:
Agreed